src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -307,6 +307,10 @@ public override Binder VisitAccessorDeclaration(AccessorDeclarationSyntax parent
                         if ((object)accessor != null)
                         {
                             resultBinder = new InMethodBinder(accessor, resultBinder);
+                            if (propertyOrEventDecl.IsKind(SyntaxKind.PropertyDeclaration))
+                            {
+                                resultBinder = new FieldKeywordBinder(accessor, resultBinder);
```

Comment:
>new FieldKeywordBinder(accessor, resultBinder) [](http://example.com/codeflow?start=47&length=46)

Will it be safe to place this binder "above" the ```InMethodBinder``` in case somewhere we expect an ```InMethodBinder``` to be returned by the factory?

Reply:
Moved it above.